### DISCUSSION TITLE
[Suggestion] Merge GB and GBC into one console

### DISCUSSION THREAD

**Discussion by Blazekickn** [RetroAchievements/RAWeb#762] [Ideas]
It makes sense but its useless for ra. Only really gets in the way when trying to link, for example, color hacks of gb games to their original sets.

**Comment by rzumer** [comment]
Another user pointed out the same issue with commercial re-releases that could otherwise be linked into a single entry, like Wario Land II.

Other platforms that cover multiple hardware configurations/variants (e.g. PC-8801, Apple II, Oric) use a single ID, so merging GB/GBC would simplify the system and be consistent with other decisions.

It might require changes to the emulators, so I tagged it as hybrid.

**Comment by kdecks** [comment]
It could be a complex problem, but it's definitely worth it on the long term.

**Comment by Blazekickn** [comment]
bringing this up again. I dont see how this is a "complex problem" considering id imagine it being a literal one number change to get integration to link entries to console 4 instead of console 6 and someone with site access can already change an entries console id.

**Comment by Jamiras** [comment]
#793 is specifically written to allow linking hashes across consoles. I think that's a better solution than trying to merge the consoles at the database level.
